sdk/python/pulumi_kubernetes/tables.py

```diff
@@ -643,7 +198,7 @@
     "ip_family": "ipFamily",
     "iscsi_interface": "iscsiInterface",
     "job_template": "jobTemplate",
-    "json_path": "jsonPath",
+    "json_path": "JSONPath",
```

Comment:
Why did this change?

Reply:
Oh interesting. This seems to be coming from the schema. `kubernetes:apiextensions.k8s.io/v1beta1:CustomResourceColumnDefinition` has a property "JSONPath". But it was updated to "jsonPath" in v1 `kubernetes:apiextensions.k8s.io/v1:CustomResourceColumnDefinition`.

Not sure what to do here...

Reply:
Oof. Yeah, I don't think we've had differing `camelCase` names (though `JSONPath` is really `PascalCase`) that case map to the same `snake_case` name before. This... isn't great, and I don't think our current design can accommodate it. We should check to see if the case mapping table is at least deterministic, as that may afford us some options here.

Reply:
Okay yeah the case mapping table is deterministic. We're iterating over an array in the same order so the resulting tables always ends up with the case it encountered last. What options do you mean @pgavlin ?

Reply:
This fix will deal with the immediate issue: https://github.com/pulumi/pulumi/pull/4907

I opened an issue to track the underlying problem: https://github.com/pulumi/pulumi/issues/4902